packages/wonder-blocks-grid/components/flex-cell.js

```diff
@@ -54,35 +45,43 @@ type Props = {|
  * grid sizes. If you specify the `small`, `medium`, or `large`
  * props then the component will only be shown at those grid sizes.
  */
-class FlexCell extends React.Component<Props> {
+export default class FlexCell extends React.Component<Props> {
     static defaultProps = {
-        small: false,
-        medium: false,
-        large: false,
+        small: true,
+        medium: true,
+        large: true,
+        mdOrLarger: true,
+        mdOrSmaller: true,
     };
 
     static shouldDisplay(props: Props, mediaSize: MediaSize) {
         return matchesSize(props, mediaSize);
     }
 
     render() {
-        const {children, style, mediaSize, mediaSpec} = this.props;
+        const {children, style} = this.props;
 
-        if (!FlexCell.shouldDisplay(this.props, mediaSize)) {
-            return null;
-        }
+        return (
+            <Layout>
+                {({mediaSize, mediaSpec}) => {
+                    if (!FlexCell.shouldDisplay(this.props, mediaSize)) {
+                        return null;
+                    }
 
-        let contents = children;
+                    let contents = children;
```

Comment:
`let` feels like a smell to me in many cases.

For example, this can be `const` if the subsequent code is a function, like this.


```
const contents = _getContents(mediaSize, mediaSpec);
```

```
_getContents = (mediaSize, mediaSpec) => {
    const {children} = this.props;
    if (typeof children === "function") {
        const {totalColumns} = mediaSpec[mediaSize];
        return children({mediaSize, totalColumns});
    }

    return children;
};
```

This would make things cleaner, I think. What do you think?